autogen/main/variables.tf.tmpl

```diff
@@ -251,6 +251,11 @@ variable "cluster_autoscaling" {
     max_cpu_cores       = number
     min_memory_gb       = number
     max_memory_gb       = number
+    gpu_resources = list(object({
```

Comment:
the tf fmt lint error is possibly due to this
```suggestion
    max_memory_gb       = number
    gpu_resources           = list(object({
```

Reply:
I tried that originally but no luck. I've now moved the object all on one line like other object definitions.